patina_dxe_core/src/gcd/spin_locked_gcd.rs

```diff
@@ -435,9 +436,6 @@ impl GCD {
             capabilities |= efi::MEMORY_ISA_VALID;
         }
 
```

Comment:
the io_blocks portion of the GCD does the same thing, should you update that too?

e.g. `add_io_space` will also call `self.init_io_blocks()` if `io_blocks.capacity() == 0`